jest.config.js

```diff
@@ -13,6 +13,14 @@ module.exports = {
         '!**/__stories__/**/*',
         '!**/*/*.stories.{ts,tsx}',
     ],
+    coverageThreshold: {
```

Comment:
I'm not sure we need coverage tests on CI. These metrics are inconsistent.

Reply:
If we can automate it - why not?

Reply:
Because it can break thresholds even if you add code where tests are not applicable or not critical. Such CI fails will be a pain for a PR's author.

Reply:
> > It looks like we don't have culture to update/write new tests on PRs. It's up to help this practice.
> 
> Maybe it's better to integrate this aspect of culture in some sort of guidelines? And gently enforce it by maintainers during PR reviews — they should know what tests should really be added / updated in each PRs.

Are there any suitable guidelines for maintainers/contributors? The last one contains legal information.

Reply:
@helenjer We can add a new job for CI that will place comment with coverage changes in PRs using Github Action.